CHANGELOG.md

```diff
@@ -1,3 +1,66 @@
+# 1.0.10-rc1 (2020-06-23)
+## Edge Agent
```

Comment:
@varunpuranik please review this release note plus my comments.

Reply:
@varunpuranik where did 1.0.9 changelog go?

Reply:
@philipktlin - which comments are you specifically referring to? 
@massand - this one?
https://github.com/Azure/iotedge/blob/release/1.0.9/CHANGELOG.md#109-2020-03-18

Reply:
@varunpuranik all my comments are addressed. btw for 1.0.9 release, I think we should combine all changes in all 1.0.9 rc release notes into 1.0.9 release notes, what do you think?